pkg/operator/ceph/controller/spec.go

```diff
@@ -106,11 +106,12 @@ PERIODICITY=%s
 LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
 LOG_MAX_SIZE=%s
 ROTATE=%s
+ADDITIONAL_LOG_FILES=%s
 
 # edit the logrotate file to only rotate a specific daemon log
 # otherwise we will logrotate log files without reloading certain daemons
 # this might happen when multiple daemons run on the same machine
-sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"
+sed -i "s|*.log|$CEPH_CLIENT_ID.log $ADDITIONAL_LOG_FILES|" "$LOG_ROTATE_CEPH_FILE"
```

Comment:
Wanted to make sure, `logrotate.conf` works fine with multiple file name  with a space

```
/var/log/rgw.log rgwops.log{
    weekly
    ....
    ...  
  }

```
I think full filename should be provided for both the files

```
/var/log/rgw.log /var/log/rgwops.log{
    weekly
    ....
    ...  
  }

```

cc @BlaineEXE @ideepika